### ISSUE TITLE
Data splits for BDD100k

### ISSUE THREAD

**Issue by tonmoy-saikia** [bethgelab/robust-detection-benchmark#2]
Hi! Has the train/val/test splits for BDD100K been released? I couldn't find it in the repo.

**Comment by michaelisc** [comment]
I am not 100% sure what you mean. BDD100k has a pre-defined train/val split that we used. Regarding our weather/daytime splits we simply used the existing weather/daytime annotations to further split the training and validation splits. Does that answer your question?

**Comment by tonmoy-saikia** [comment]
Sorry for the lack of clarity. Yes, I was referring to the weather/daytime splits. In Appendix C of the paper you mention: 
> For testing, we created 3 subsets of the validation set each containing 725 images in clear, rainy or snowy conditions..

I just wanted to know which 725 images were chosen for each case.

**Comment by michaelisc** [comment]
Oh, you are right! We used the first 725 images of each split. 

Code example:
```
# assert weather choice
assert weather in ['clear' 'foggy' 'overcast' 'partly cloudy' 'rainy' 'snowy' 'undefined']

# load data
val_labels = json.load(open(os.path.join(label_dir, 'bdd100k_labels_images_val.json')))

# filter by weather
val_labels = [x for x in val_labels if x['attributes']['weather'] == weather]

# remove images without annotations
val_labels = [x for x in val_labels if len x['labels'] != 0]

# select first 725
val_labels = val_labels[:725]
```

**Comment by tonmoy-saikia** [comment]
Thanks!
